src/main/resources/templates/db_scripts/defaultFileExtensions.sql

```diff
@@ -283,7 +283,7 @@ VALUES
       "extension":".xml",
       "description": "",
       "dataTypes":[
-        "MARC"
+        "MARC_BIB"
```

Comment:
Maybe there should be more dataTypes? MARC_AUTH and MARC_HOLDINGS?

Reply:
And I assume that this script is already part of envs, so you need to do migration.